LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/JoinRequestMessageHandler.cs

```diff
@@ -98,7 +98,7 @@ internal async Task ProcessJoinRequestAsync(LoRaRequest request)
                     return;
                 }
 
-                if (!joinReq.CheckMic(loRaDevice.AppKey))
+                if (!joinReq.CheckMic(loRaDevice.AppKey.Value))
```

Comment:
nit: I would cache it. We keep on repeating loRaDevice.AppKey.Value

Reply:
Done in a77060bca2caa825e0a345fdb02250e3ddad242e